ml-operator/templates/ml-operator.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: {{ template "apiVersion.Deployment" . }}
+kind: Deployment
+metadata:
+  name: {{ include "ml-operator.name" . }}
+  labels:
+    {{- include "ml-operator.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1 # exeed the replicas by 1 pod
+  selector:
+    matchLabels:
+      {{- include "ml-operator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "ml-operator.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: mojaloop-operator
+      containers:
+        {{- if index .Values "ml-operator" "enabled" }}
+        - name: ml-operator
+          image: {{ index .Values "ml-operator" "image"  }}
+          # command: [ "tail", "-f", "/dev/null" ]
+          volumeMounts:
+            - name: ml-operator-config-volume
+              mountPath: /opt/ml-operator/config
+          env:
+            - name: SLACK_WEBHOOK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: ml-operator-secrets
```

Comment:
Actually I would recommend making this configurable

Reply:
Yeah good point